### PR TITLE
barista: Added icon page layout and adjusted color-wheel component

### PR DESCRIPTION
**Pull request by lara-aigmueller**
### <strong>Pull Request</strong>

#related to #231 

<hr>
Hi, thank you for contributing to the Dynatrace Angular Component library (PR).

To ensure a fast process and merging of your PR please make sure it fulfills the
coding standards and contribution guidelines.

- A feature proposal has been provided, discussed and approved first.
- There is a meaningful description of the issue in GitHub (Screenshots are
  often helpful).
- If the PR introduces breaking-changes or deprecations it matches the following
  guidelines.
  - The commit message follows our commit guidelines.
  - Tests for the changes have been added (for bug fixes / features).
  - Docs have been added / updated (for bug fixes / features).

Please fill out the checkbox below: <br>

#### Type of PR

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or change that would cause existing functionality to
      not work as expected)
- [ ] Documentation update (changes to documentation)
- [ ] Other (if none of the above apply)

#### Checklist

- [ ] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: apps/barista/src/components/icon-color-wheel/icon-color-wheel.ts [hunk 1]
```diff
@@ -40,6 +41,16 @@ let uniqueColorwheelId = 0;
...
 
+  @Input()
+  get iconsvg(): string {
+    return this._iconsvg.toString();
+  }
+  set iconsvg(value: string) {
+    this._iconsvg = this._sanitizer.bypassSecurityTrustHtml(value);
+  }
+  /** @internal Icon SVG string as safe HTML */
+  _iconsvg: SafeHtml;
```
Comment (tomheller) [inline apps/barista/src/components/icon-color-wheel/icon-color-wheel.ts]: Could this member be private and we use the getter in the template?
```suggestion
  private _iconsvg: SafeHtml;
```

File: apps/barista/src/components/icon-color-wheel/icon-color-wheel.ts [hunk 2]
```diff
@@ -40,6 +41,16 @@ let uniqueColorwheelId = 0;
 export class BaIconColorWheel {
   @Input() iconname: string;
 
+  @Input()
+  get iconsvg(): string {
```
Comment (tomheller) [inline apps/barista/src/components/icon-color-wheel/icon-color-wheel.ts]: NIT: Please add a comment to the public input.

**Comment by thomaspink** [comment]
In my opinion we don't need this page, because it can be part of the normal single-page and we just ship the content. The icon rendering itself is much like the color component and grid just a component that will spin up in the content. This also has the benefit that we can prerender the changelog in the builder.

**Comment by lara-aigmueller** [comment]
Not needed, icon pages will use the single page template as discussed.

**Pull request closed without merging**